### ISSUE TITLE
Playbook Template "Disable malicious inbox rule" was deleted.

### ISSUE THREAD

**Issue by Mela-SAG** [microsoft/Microsoft-Cloud-App-Security#4]
Hi 

It look like the Playbook Template "Disable malicious inbox rule" was deleted, could you please upload it again?

Best regards
Mela

**Comment by CraigCloudITPro** [comment]
yes i would also like to have a copy of this playbook, please can you re-upload it, thanks
